<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Account_Label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>账号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password_Label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Account_LineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="Register_Label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>还没有账号？快来注册一个</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_LineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeVerCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
